suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+    rabbitmq_var_log="/var/log/rabbitmq"
+    rabbitmq_report_log=$rabbitmq_var_log/rabbitmq_report.log
     source /etc/rabbitmq/rabbitmq-env.conf
-    pushd `pwd`
     cd /var/lib/rabbitmq
     su rabbitmq -s /bin/sh -c "/usr/lib64/rabbitmq/bin/rabbitmqctl report" > ${rabbitmq_report_log}
-    popd
-    find_and_plog_files_0 ${rabbitmq_tmpdir} -type f
-    rm -rf ${rabbitmq_tmpdir}
+    cd -
+    find_and_plog_files_0 ${rabbitmq_var_log} -type f
```

Comment:
Because we would like to include all log files in `/var/log/rabbitmq` from rabbitmq for example `rabbit@crowbar.log  rabbit@crowbar-sasl.log  startup_err  startup_log`